deploy/crds/chaosengine_crd.yaml

```diff
@@ -28,6 +29,60 @@ spec:
           type: object
         spec:
           type: object
+          properties:
+            monitoring: 
+              type: boolean
+            jobCleanUpPolicy: 
+              type: string
+              pattern: ^(delete|retain)$
+              # alternate ways to do this in case of complex pattern matches
+              #oneOf:
+              #  - pattern: '^delete$'
+              #  - pattern: '^retain$'
+            appinfo:
+              type: object
+              properties:
+                appkind: 
+                  type: string
+                  pattern: ^(deployment|statefulset|daemonset)$
+                applabel: 
+                  type: string
+                appns: 
+                  type: string
+            chaosServiceAccount:
+              type: string
+            components:
+              type: object
+              properties: 
+                monitor: 
+                  type: object
+                  properties: 
+                    image: 
+                      type: string
+                runner: 
+                  type: object
+                  properties:
+                    image: 
```

Comment:
Should we also have a `type` field here to support `goExecutor`???

Reply:
Yes. Thanks for pointing out. Adding this

Reply:
Have added the validation for runner type!

Reply:
Thanks @ksatchit